cms/djangoapps/contentstore/tasks.py

```diff
@@ -606,23 +608,17 @@ def read_chunk():
             static_content_store=contentstore(),
             target_id=courselike_key,
             verbose=True,
-            status=self.status
```

Comment:
Why have we removed this? We are not keeping error messages in the Artifact object now?

Reply:
We raising an exception and catching that in the `tasks.py` this has enabled us to remove the dependency of `status` object for `import_course_for_xml`